api/src/main/java/org/motechproject/nms/api/web/KilkariController.java

```diff
@@ -101,4 +104,73 @@ public void saveInboxCallDetails(@RequestBody InboxCallDetailsRequest request) {
         //todo: the work...
     }
 
+    /**
+     * 4.2.3
+     * Create Subscription
+     *
+     * IVR shall invoke this API to create a new Kilkari subscription
+     *
+     */
+    @RequestMapping(value = "/subscription",
+            method = RequestMethod.POST,
+            headers = { "Content-type=application/json" })
+    @ResponseStatus(HttpStatus.OK)
+    public void createSubscription(@RequestBody SubscriptionRequest subscriptionRequest)
+        throws NotFoundException {
```

Comment:
Remember Frank said the throws can be removed if NotFoundException extends IllegalStateException